app/api/v1/views/office_views.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python3
+""" Routes Office Information """
+
+from flask import Blueprint, jsonify, request
+from app.api.v1.models.office_models import PoliticalOffices
+from app.api.v1.views.response_vars import (
+    more_data_fields_response, few_data_fields_response,
+    id_out_of_range_response, id_cannot_be_zero_response
+)
+
+OFFICE_BP_V1 = Blueprint("v1_office", __name__, url_prefix="/api/v1")
+
+
+@OFFICE_BP_V1.route("/offices", methods=["GET"])
```

Comment:
You can reduce the complexity by creating a list of methods, for instance, GET, DELETE and PATCH use ids, then match function to either action using an `if`

Reply:
Hi, I've refactored view. I've added a helper function to perform repeated tasks